security-analysis/src/main/java/com/powsybl/security/Security.java

```diff
@@ -160,33 +163,49 @@ public static String printLimitsViolations(List<LimitViolation> violations) {
     public static String printLimitsViolations(List<LimitViolation> violations, LimitViolationFilter filter) {
         Objects.requireNonNull(violations);
         Objects.requireNonNull(filter);
+
+        TableFormatterFactory formatterFactory = new AsciiTableFormatterFactory();
+        Writer writer = new StringWriter();
+        TableFormatterConfig formatterConfig = new TableFormatterConfig(Locale.getDefault(), ',', "inv", true, true);
         List<LimitViolation> filteredViolations = filter.apply(violations);
-        if (filteredViolations.size() > 0) {
-            filteredViolations.sort(Comparator.comparing(LimitViolation::getSubjectId));
-            Table table = new Table(9, BorderStyle.CLASSIC_WIDE);
-            table.addCell("Country");
-            table.addCell("Base voltage");
-            table.addCell("Equipment (" + filteredViolations.size() + ")");
-            table.addCell("Violation type");
-            table.addCell("Violation name");
-            table.addCell("Value");
-            table.addCell("Limit");
-            table.addCell("abs(value-limit)");
-            table.addCell("Loading rate %");
-            for (LimitViolation violation : filteredViolations) {
-                table.addCell(violation.getCountry() != null ? violation.getCountry().name() : "");
-                table.addCell(Float.isNaN(violation.getBaseVoltage()) ? "" : Float.toString(violation.getBaseVoltage()));
-                table.addCell(violation.getSubjectId());
-                table.addCell(violation.getLimitType().name());
-                table.addCell(getViolationName(violation));
-                table.addCell(Float.toString(violation.getValue()));
-                table.addCell(getViolationLimit(violation));
-                table.addCell(Float.toString(Math.abs(violation.getValue() - violation.getLimit() * violation.getLimitReduction())));
-                table.addCell(Integer.toString(getViolationValue(violation)));
-            }
-            return table.render();
+
+        try (TableFormatter formatter = formatterFactory.create(writer,
+                "",
+                formatterConfig,
+                new Column("Country"),
+                new Column("Base voltage"),
+                new Column("Equipment (" + filteredViolations.size() + ")"),
+                new Column("Violation type"),
+                new Column("Violation name"),
+                new Column("Value"),
```

Comment:
Another idea is to configure the CellStyle here: all the cells of the same column should be rendered as the same way